src/session.coffee

```diff
@@ -28,6 +28,7 @@ define [
 
         connect: (shop_code)->
           @shop_code = shop_code
+          @plugins_manager.session = @
```

Comment:
We should add a test about this one.
e.g on connect session is being passed to plugins_manager